react/StoreLink.tsx

```diff
@@ -53,7 +53,12 @@ export const defaultButtonProps: ButtonProps = {
   size: 'regular',
 }
 
-const CSS_HANDLES = ['link', 'label', 'childrenContainer', 'buttonLink']
+const CSS_HANDLES = [
+  'link',
+  'label',
+  'childrenContainer',
+  'buttonLink',
+]
```

Comment:
Uma vez que voce não tem mais o `linkContainer`.

```suggestion
const CSS_HANDLES = ['link', 'label', 'childrenContainer', 'buttonLink']
```